Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
Is it necessary ? Moreover, isn't the "identifier" coupled to the storage strategy ? I think that right know it's an useless feature that adds another thing to maintain (until, maybe, it's demonstrated that's really useful in specific cases)